cmd/skipper/main.go

```diff
@@ -30,9 +30,9 @@ package main
 
 import (
 	"flag"
+	log "github.com/Sirupsen/logrus"
```

Comment:
We did, to some extent. We're aware of that warning and this was an informed decision